.github/actions/build/action.yaml

```diff
@@ -0,0 +1,64 @@
+name: 'Build'
+description: 'Build images'
+inputs:
+  docker_password:
+    description: 'Docker password'
+    required: true
+  repository_owner:
+    description: 'Repository owner'
+    required: true
+  github_token:
+    description: 'GitHub Token'
+    required: true
+  labels:
+    description: 'Labels added on metadata'
+    required: false
+    default: ''
+runs:
+  using: "composite"
+  steps:
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY_IMAGE }}    
+          labels: |
+            {{ inputs.labels }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3.0.0
+        with:
+          username: teslamate
+          password: ${{ inputs.docker_password }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ inputs.repository_owner }}
+          password: ${{ inputs.github_token }}
+      - name: Build and push by digest
+        id: build
+        uses: docker/build-push-action@v5.0.0
+        with:
+          context: .
+          platforms: ${{ matrix.platform }}
+          labels: ${{ steps.meta.outputs.labels }}
+          tags: ${{ steps.docker_meta.outputs.tags }}
+          cache-from: type=registry,ref=ghcr.io/teslamate-org/teslamate:buildcache-${{ matrix.cache_id }}
+          cache-to: type=registry,ref=ghcr.io/teslamate-org/teslamate:buildcache-${{ matrix.cache_id }},mode=max
+          outputs: type=image,name=${{ env.REGISTRY_IMAGE }},push-by-digest=true,name-canonical=true,push=true
+      - name: Export digest
+        shell: bash      
+        run: |
+          mkdir -p /tmp/teslamate/digests
+          digest="${{ steps.build.outputs.digest }}"
+          touch "/tmp/teslamate/digests/${digest#sha256:}"
+          echo ${{ env.DOCKER_METADATA_OUTPUT_VERSION }} > /tmp/teslamate/version
+      - name: Upload digest
+        uses: actions/upload-artifact@v3
+        with:
+          name: digests
+          path: /tmp/teslamate/*
```

Comment:
It should be /tmp/teslamate/digests/*

Reply:
I finished to refactoring. We finally stay on /tmp/digest.